daemon/mgr/container_logger.go

```diff
@@ -0,0 +1,49 @@
+package mgr
+
+import (
+	"github.com/alibaba/pouch/apis/types"
+	"github.com/alibaba/pouch/daemon/containerio"
+	"github.com/alibaba/pouch/daemon/logger"
+
+	"github.com/sirupsen/logrus"
+)
+
+func optionsForContainerio(c *Container) []func(*containerio.Option) {
```

Comment:
This function is only used for log options and in the file `container_logger.go`

Maybe the name `logOptionsForContainerio` is more appropriate?

Reply:
make senses.